packages/fxa-payments-server/server/lib/server.js

```diff
@@ -175,12 +175,12 @@ module.exports = () => {
       CLIENT_CONFIG,
       {}
     );
-    for (const route of INDEX_ROUTES) {
```

Comment:
The upgraded eslint seems to require this: Using the `for` loop is rejected with an error saying the variable `routes` is unused. Switching to `forEach` seems to fix this while doing essentially the same thing 😕